using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -229,7 +279,11 @@ If the provided URL is null or empty, it is returned as is. Otherwise, the follo
 public static void appendVisitorInfoForURL(final String baseURL, final AdobeCallback<String> callback);
 ```
 
-#### **Example**
+o  *baseUrl* is the URL to which the visitor info needs to be appended. Returned as is if it is nil or empty.
```

Comment:
- Move the parameters to after the code sample.
- Spell out "info" to "information".
- Reword the 2nd sentence to read: "If the visitor information is nil or empty, the URL is returned as is." and move to the next line.